loomi/regulariza-morador-client
# Feature/login integration

## Login Integration

---

**What I did:**

- Integrate Login with API

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Add `REACT_APP_MOCK_URL=https://stage.api.reg.loomi.com.br` to the `env` file.
3. Try to login with `10850753465` and `NovaSenha@123`. You can create another account with Swagger if you want to.
4. Tip: check if cookies are correct, try to delete all cookies, delete one specific cookie. Mess with whatever you want to see if you will be logout.
